projects/plugins/jetpack/modules/subscriptions.php

```diff
@@ -620,17 +628,44 @@ public function setting_comment_follow() {
 		echo '<p><span class="description">' . esc_html__( 'Introduction text sent when someone follows a post on your blog. (Site and confirmation details will be automatically added for you.)', 'jetpack' ) . '</span></p>';
 	}
 
+	/**
+	 * HTML output helper for Welcome section.
+	 */
+	public function setting_welcome() {
+		$settings = $this->get_settings();
+		echo '<textarea name="subscription_options[welcome]" class="large-text" cols="50" rows="5">' . esc_textarea( $settings['welcome'] ) . '</textarea>';
+		echo '<p><span class="description">' . esc_html__( 'Welcome text sent when someone follows your blog.', 'jetpack' ) . '</span></p>';
+	}
+
 	/**
 	 * Get default settings for the Subscriptions module.
 	 */
 	public function get_default_settings() {
 		$site_url    = get_home_url();
 		$display_url = preg_replace( '(^https?://)', '', untrailingslashit( $site_url ) );
 
+		// TODO: Remove the locale check once the translation is ready.
+		$is_english_locale = str_starts_with( get_locale(), 'en' );
+		if ( $is_english_locale ) {
```

Comment:
I don't quite understand this. Since both strings are new and untranslated right now, what's the benefit in having this condition?

Reply:
I was wrongfully assuming that it would use the translation already present on WPCOM when deployed there. I'll fix this.